markdown/org/docs/patterns/bee/options/bellaguide/en.md

```diff
@@ -4,6 +4,14 @@ title: Show Bella
 
 Shows the outline of the Bella block Bee is based on
 
+This is to help give context to the other options of Bee especially when using the fit options
```

Comment:
This is super cool. I hadn't seen options like this before.

Reply:
Thank you :) 
Yea it evolved from the guide that I was using originally to code Bee and I kinda felt like it should remain as Bee is so different from Bella that the options might not make much sense without it.